pkgs/by-name/tp/tpm2-openssl/package.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
+, autoreconfHook
+, fetchFromGitHub
+, gitUpdater
+, autoconf-archive
+, pkg-config
+, openssl
+, tpm2-tss
+}: stdenv.mkDerivation (finalAttrs: {
+  pname = "tpm2-openssl";
+  version = "1.2.0";
+  src = fetchFromGitHub {
+    owner = "tpm2-software";
+    repo = "tpm2-openssl";
+    rev = finalAttrs.version;
+    hash = "sha256-mZ4Z/GxJFwwfyFd1SAiVlQqOjkFSzsZePeuEZtq8Mcg=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    autoconf-archive
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+    tpm2-tss
+  ];
+
+  preConfigurePhases = [
+    "setConfigureFlags"
+  ];
+
+  setConfigureFlags = ''
+    export configureFlags="$configureFlags --with-modulesdir=$out/lib/ossl-modules"
+  '';
+
+  prePatch = ''
+    echo ${finalAttrs.version} > VERSION
+  '';
+
+  passthru.updateScript = gitUpdater { };
```

Comment:
I see @r-ryantm updating packages that don't have this. I mostly see this `gitUpdater` used for packages with self-hosted git repositories. Is this needed, or can it be removed?

Reply:
Thanks :) I removed it in c12cd7ea273caf8fe9a3daefe7e5babf27d18e9f